aliyun/src/main/java/org/apache/iceberg/aliyun/AliyunProperties.java

```diff
@@ -25,21 +25,74 @@
 import org.apache.iceberg.util.PropertyUtil;
 
 public class AliyunProperties implements Serializable {
+  /**
+   * The domain name used to access OSS. OSS uses HTTP Restful APIs to provide services. Different regions are accessed
+   * by using different endpoints. For the same region, access over the internal network or over the Internet also uses
+   * different endpoints. For more information, see:
+   * https://www.alibabacloud.com/help/doc-detail/31837.htm
+   */
+  public static final String OSS_ENDPOINT = "oss.endpoint";
+
+  /**
+   * Aliyun uses an AccessKey pair, which includes an AccessKey ID and an AccessKey secret to implement symmetric
+   * encryption and verify the identity of a requester. The AccessKey ID is used to identify a user.
+   * <p>
+   * For more information about how to obtain an AccessKey pair, see:
+   * https://www.alibabacloud.com/help/doc-detail/53045.htm
+   */
+  public static final String ACCESS_KEY_ID = "access.key.id";
+
+  /**
+   * Aliyun uses an AccessKey pair, which includes an AccessKey ID and an AccessKey secret to implement symmetric
+   * encryption and verify the identity of a requester.  The AccessKey secret is used to encrypt and verify the
+   * signature string.
+   * <p>
+   * For more information about how to obtain an AccessKey pair, see:
+   * https://www.alibabacloud.com/help/doc-detail/53045.htm
+   */
+  public static final String ACCESS_KEY_SECRET = "access.key.secret";
```

Comment:
Is this a standard name? Why so much nesting?

Reply:
The hadoop aliyun 's configuration keys are ( see: https://hadoop.apache.org/docs/stable/hadoop-aliyun/tools/hadoop-aliyun/index.html)
* fs.oss.accessKeyId
* fs.oss.accessKeySecret
* fs.oss.credentials.provider

As for peole who use the aliyun oss to storage the iceberg files, the class name `OSSFileIO` has implied the oss distribution file system.   So in theory the `accessKeyId` and `accessKeySecret` are enough to express their meanings.  Consider the iceberg naming style,  `access-key.id` and `access-key.secret` may be better names ?

Reply:
Another background,  I did some research on the Chinese community and few feedback from Chinese users.  Almost all of them has built their iceberg tables on top of the aliyun OSS  by forking this PR: https://github.com/apache/iceberg/pull/2230. That means they has used the `access.key.secret` and `access.key.id` as their configuration keys in their production env.  

If we plan to change the apache iceberg official keys to `access-key.id` and `access-key.secret`,  I think this will break their pipelines if their migrate their forked iceberg repo to apache iceberg release.  I'm thinking maybe it's okay to keep the original configuration keys ?  FYI @rdblue .

Reply:
It's fine to keep them if we think it will break people, but in the future we should be careful. Being lax about naming leads to poor user experience.

Reply:
> Being lax about naming leads to poor user experience.

Agree.  will be more careful about those names in future.